### ISSUE TITLE
Dependency issue

### ISSUE THREAD

**Issue by dmehring** [radio-astro-tools/uvcombine#24]
I've installed casa_formats_io, but upon

`import uvcombine`

I get (long stack trace omitted)

`ModuleNotFoundError: No module named 'casa_formats_io._casa_chunking`

Any ideas?

**Comment by keflavich** [comment]
`uvcombine` is a bit out of date.  Could you post the full traceback?  I'll try to debug it.

**Comment by dmehring** [comment]
Thanks. Here you go

```
(xenv) 16:28 dmehring@eowyn:casa-formats-io[py3.6](main):$ ipython
Python 3.6.12 (default, Sep 15 2020, 12:49:50) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.16.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import uvcombine
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-4a12ef56ffe1> in <module>
----> 1 import uvcombine

/export/data_1/dmehring/casa_eclipse/projects/casa6/casatools/testing/xenv/lib64/python3.6/site-packages/uvcombine-0.0.dev120-py3.6.egg/uvcombine/__init__.py in <module>
     13 # For egg_info test builds to pass, put package imports here.
     14 if not _ASTROPY_SETUP_:
---> 15     from .uvcombine import (feather_plot, feather_simple, spectral_regrid,
     16                             spectral_smooth_and_downsample,
     17                             fourier_combine_cubes)

/export/data_1/dmehring/casa_eclipse/projects/casa6/casatools/testing/xenv/lib64/python3.6/site-packages/uvcombine-0.0.dev120-py3.6.egg/uvcombine/uvcombine.py in <module>
     10 import radio_beam
     11 from reproject import reproject_interp
---> 12 from spectral_cube import SpectralCube
     13 from spectral_cube import wcs_utils
     14 from astropy.io import fits

/export/data_1/dmehring/casa_eclipse/projects/casa6/casatools/testing/xenv/lib64/python3.6/site-packages/spectral_cube-0.0.0-py3.6.egg/spectral_cube/__init__.py in <module>
     14 
     15 # Import the following sub-packages to make sure the I/O functions are registered
---> 16 from .io import casa_image
     17 del casa_image
     18 from .io import class_lmv

/export/data_1/dmehring/casa_eclipse/projects/casa6/casatools/testing/xenv/lib64/python3.6/site-packages/spectral_cube-0.0.0-py3.6.egg/spectral_cube/io/casa_image.py in <module>
     12 from .. import wcs_utils
     13 
---> 14 from casa_formats_io import getdesc, coordsys_to_astropy_wcs, image_to_dask
     15 
     16 # Read and write from a CASA image. This has a few

/export/home/eowyn/dmehring/casa_eclipse/projects/casa6/casatools/radio-astro-tools/casa-formats-io/casa_formats_io/__init__.py in <module>
----> 1 from .casa_dask import *  # noqa
      2 from .casa_low_level_io import *  # noqa
      3 from .casa_wcs import *  # noqa
      4 from .version import version as __version__

/export/home/eowyn/dmehring/casa_eclipse/projects/casa6/casatools/radio-astro-tools/casa-formats-io/casa_formats_io/casa_dask.py in <module>
     11 
     12 from .casa_low_level_io import Table
---> 13 from ._casa_chunking import _combine_chunks
     14 
     15 __all__ = ['image_to_dask']

ModuleNotFoundError: No module named 'casa_formats_io._casa_chunking'

```

**Comment by keflavich** [comment]
OK looks like this issue is actually bigger and is really a `casa_formats_io` issue. I'll x-post it here.

(you should get the same issue if you do `from spectral_cube import SpectralCube`)

**Comment by astrofrog** [comment]
@dmehring - how did you install casa-formats-io? From the log it looks like casa_formats_io is not inside the site-packages, which makes me think that somehow it is importing from a clone of the repo as opposed to a pip installed version of the package?

**Comment by dmehring** [comment]
Unfortunately, I no longer remember the details of that install, but it seems likely that I indeed didn't install using pip (because I didn't realize that I could), so cloned and tried to build via setup.py. I tried today installing via pip all the packages, including radio_astro_tools, that a build of uvcombine complained it was looking for. All those packages installed without incident and importing uvcombine inside python worked cleanly. So, indeed, the issue seems to be that I didn't install using pip, and problem solved.
Thanks!
